src/device.c

```diff
@@ -1505,9 +1485,60 @@ void uvc_process_status_xfer(uvc_device_handle_t *devh, struct libusb_transfer *
                     event,
                     selector,
                     attribute,
-                    data, data_len,
+                    content, content_len,
```

Comment:
It makes sense to strip off the first 5 bytes when delivering this update, but 1) this breaks any existing code that might be relying on the update's content and 2) the originator ID is lost.

I'd be open to adding an `originator` parameter to the callback type, maybe between `status_class` and `event`, and also making the 5-byte-offset change. That way, anything that uses the callback will break, but at least the breakage will happen at compile time.